tools/collect_archive_logs.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# save directory from which command is initiated
+collect_logs_directory=$(pwd)
```

Comment:
This goes back to a previous comment I made in a commit, but with this line, we should either:

- Make sure to include in the `README.md` a mention to run this from the same directory the script is in
- Change this to the following:

```bash
# make the vmdb/log directory the current directory
vmdb_logs_directory="/var/www/miq/vmdb"
pushd ${vmdb_logs_directory}

# save directory from which command is initiated
collect_logs_directory=${vmdb_logs_directory}/tools
```

Or if you move these scripts into a subdir, `${vmdb_logs_directory}/tools/[my_new_subdir]`.

Reply:
If you do go the "directory route", I would suggest something like:

```
/var/www/miq/vmdb/
└── tools/
    └── collect_logs/
        ├── archive_logs.sh
        ├── current_logs.sh
        └── README.md
```

Reply:
Hey @NickLaMuro I went the directory route so I assume this also takes care of your `exclude_files` concern as I updated the README as well.